styleguide/_layouts/default.jade

```diff
@@ -20,9 +20,52 @@ html(lang="en")
     link(href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.1/css/font-awesome.min.css" rel="stylesheet")
     script(src="https://code.jquery.com/jquery-1.12.3.min.js" integrity="sha256-aaODHAgvwQW1bFOGXMeX+pC4PZIPsvn2h1sArYOhgXQ=" crossorigin="anonymous")
 
+  .container-fluid
+    .row.row-eq-height
+      div.navZoolander-container.bg-blue-gray#navZoolander-container
+        a(href="javascript:void(0)").closebtn(onclick="closeNav()") <i class="fa fa-times white fa-2x"></i>
```

Comment:
Can't you use a colon for jade block expansion on the `<I>` tag here instead of hard coding HTML?
http://jade-lang.com/reference/tags